src/generators/swagger2.ts

```diff
@@ -0,0 +1,22 @@
+import { CodeGenOptions } from "../options/options";
+import { transformToCodeWithMustache } from "../transform/transformToCodeWithMustache";
+import { getViewForSwagger2, ViewData } from "../getViewForSwagger2";
+import { CodeGenerator } from "./codeGenerator";
+
+function verifyThatWeAreGeneratingForSwagger2(opts: CodeGenOptions): void {
+  if (opts.swagger.swagger !== "2.0") {
+    throw new Error("Only Swagger 2 specs are supported");
+  }
+}
+
+export const Swagger2Gen: CodeGenerator = {
+  getViewData: function(opts: CodeGenOptions): ViewData {
```

Comment:
>function [](start = 15, length = 8)

Personal preference: Should we convert these to arrow functions instead? that would look slightly cleaner to me here, and we could remove some redudnant typing. I.e.


````
export const Swagger2Gen: CodeGenerator = {
  getViewData: opts => {
    verifyThatWeAreGeneratingForSwagger2(opts);

    return getViewForSwagger2(opts);
  },
  getCode: opts => {
    const data = this.getViewData(opts);
    return transformToCodeWithMustache(data, opts.template, opts.mustache);
  }
};
````

Reply:
Agree, that makes it cleaner and the scope easier to follow.

Reply:
@mtennoe I have updated it now.

Reply:
@erictuvesson - Great!
There seems to be some minor build issues though that needs to be addressed